codice/ddf
# [2.16.x]DDF-5131 moved session lifetime setting to the session factory

#### What does this PR do?
Backport (not a clean cherry-pick) of https://github.com/codice/ddf/pull/5150
Without this, there's the possibility of getting a session that gets the default Jetty session timeout (5 minutes) instead of the configured time.
#### Who is reviewing it? 
@austinsteffes @Kjames5269 @tyler30clemens 
#### Ask 2 committers to review/merge the PR and tag them here.
@blen-desta
@stustison 

#### How should this be tested?
Configure AdminUI->Security->Session->Session Timeout and set it to 10 minutes.
Configure AdminUI->Platform->Platform UI-> Session Timeout and set it to 8 minutes
Go inactive and verify that a modal window pops up after 7 minutes.
#### Any background context you want to provide?
#### What are the relevant tickets?
Fixes: #5131 

#### Screenshots

#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
